test/cli/autogen-autoloader/test.out

```diff
@@ -51,7 +20,6 @@ Opus::Require.for_autoload(Foo::Bar::Jazz, "test/cli/autogen-autoloader/example1
 require 'in_class'
 require 'in_method'
 require 'my_gem'
-Opus::Require.on_autoload('Foo::Bar::Quuz')
```

Comment:
We remove these explicit `on_autoload` callbacks as the Zeitwerk loader allows us to insert custom callbacks as a constant is loaded. We are already calling `on_autoload` there.